finality-aleph/src/network/gossip/service.rs

```diff
@@ -200,11 +193,18 @@ impl<N: RawNetwork, AD: Data, BSD: Data> Service<N, AD, BSD> {
     fn send_to_authentication_peer(&mut self, data: AD, peer: N::PeerId) -> Result<(), SendError> {
         match self.get_authentication_sender(&peer) {
             Some(sender) => {
-                match sender.unbounded_send(data) {
+                match sender.try_send(data) {
                     Err(e) => {
                         // Receiver can also be dropped when thread cannot send to peer. In case receiver is dropped this entry will be removed by Event::NotificationStreamClosed
```

Comment:
is this comment relevant to `e.is_full()`?

Reply:
Not really, it should be only before `is_disconnected`.

Reply:
Good catch